VA-Explorer/va_explorer
# Prevent button from crashing UI by reporting error via message instead

This PR handles exceptions that arise from inability to connect to redis when running a coding algorithms async task and any similar task errors. Instead of allowing the exception to crash the UI, the exception is caught and reported to the user via the messages module.

**To test fail state:**
- Runserver and press the run coding algorithms button
- After 20 seconds of redis retrying to connect the task will fail with an error message

**To test success state:**
- `docker-compose build && docker-compose -f docker-compose.local.yml up` to start a local redis server
- Runserver and press the run coding algorithms button
- A success message should appear shortly after

Fixes #145
